### ISSUE TITLE
Can't install Sanoid on Ubuntu 24.10

### ISSUE THREAD

**Issue by AllergicToGrass** [jimsalterjrs/sanoid#967]
I'm following the steps listed for Debian distros listed [here](https://github.com/jimsalterjrs/sanoid/blob/master/INSTALL.md)

this is the error I'm receiving

```
Unpacking sanoid (2.2.0) over (2.2.0) ...
dpkg: error processing archive /home/user/sanoid_2.2.0_all.deb (--unpack):
 unable to open '/usr/lib/systemd/system/sanoid.timer.dpkg-new': No such file or directory
No apport report written because the error message indicates an issue on the local system
                                                                                         Errors were encountered while processing:
 /home/user/sanoid_2.2.0_all.deb
Notice: Download is performed unsandboxed as root as file '/home/user/sanoid_2.2.0_all.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)
Error: Sub-process /usr/bin/dpkg returned an error code (1)
root@servername:/home/user/sanoid# exit           
```

**Comment by AllergicToGrass** [comment]
I ended up using Ironicbadger's docker install to create the deb

**Comment by dmshimself** [comment]
yes I get the same error

**Comment by phreaker0** [comment]
Duplicate of #921 (and PR with fix is available here #926 )
